### DISCUSSION TITLE
This is Awesome 🤩

### DISCUSSION THREAD

**Discussion by NavBlue1** [llamafilm/tesla-http-proxy-addon#9] [Q&A] [answered]
I’ve set this up. My log looks good. 

Just need some guidance on how to integrate it with HA.

I’ve noticed “feat: Support modern vehicles using HTTP proxy #853” for the Tesla Custom integration. Should I just wait for this to be completed? 

Any expanded examples in the documentation would be appreciated. 
Thank you very much for the great work.

**Comment by llamafilm** [comment, marked as answer]
Thanks! You can install my patched version of tesla_custom integration from my fork on that PR, that’s how I’m running at home. I’m not going to provide instructions on that part because my work on that integration is still a bit rough. I’m hoping to get some help polishing it up from somebody more experienced at writing integrations.
It also depends on this PR in the underlying library: https://github.com/zabuldon/teslajsonpy/pull/453

**Reply by NavBlue1** [reply]
Legendary! Thanks for that I’ll give it a go.

By the way your instructions for setting this up were perfect. I followed them precisely and everything just worked. Nice 👍

**Reply by paky79** [reply]
@llamafilm what is the safest option to install your custom component? Should I only copy paste the tesla-custom folder? I am using the "officiale" custom Tesla addon at least to read data and I don't want to mess up everything.
